Dockerfile.eas-base

```diff
@@ -13,41 +13,35 @@ ENV NVM_VERSION='v0.39.1'
 ENV VENV_UTILS=/venv/eas-utils
 ENV UTILS_DIR=/eas/emergency-alerts-utils
 
-# Copy the bashrc file and replace existing.
+# Copy the bashrc file and remove command that exits bashrc file if not running interactively.
 RUN mv $SHELL_CONF $SHELL_CONF.bak; cat $SHELL_CONF.bak | sed 's/\[ -z "$PS1" \] && return//' > $SHELL_CONF;
 
 # Update OS to latest.
 RUN apt-get update -y
 
-# Install commonly used tools
-RUN apt-get install wget curl git unzip vim make systemctl jq apt-utils telnet dnsutils build-essential -y
-
-# Common installs some ca-certs and python related pre-reqs.
-RUN apt-get install ca-certificates -y
+# Install commonly used tools, python related pre-reqs and ca-certs
+RUN apt-get install wget curl git unzip vim make systemctl jq apt-utils telnet dnsutils build-essential \
```

Comment:
Merge this into single line as there is no terminators or seperators ; or &&

Reply:
done